frontend/src/components/pages/CreatorProfile/CreatorProfile.tsx

```diff
@@ -0,0 +1,86 @@
+import { ArrowBackIcon, EmailIcon } from "@chakra-ui/icons";
+import { Box, Button, Center, Flex, Heading, Text } from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { Link, useHistory, useParams } from "react-router-dom";
+
+import CreatorAPIClient from "../../../APIClients/CreatorAPIClient";
+import background from "../../../assets/SearchResultsBackground.png";
+import { Creator } from "../../../types/CreatorTypes";
+import LoadingSpinner from "../../common/LoadingSpinner";
+import CreatorOverview from "./CreatorOverview";
+
+interface CreatorProfileParams {
+  id: string;
+}
+
+const CreatorProfile = (): React.ReactElement => {
+  const [currentCreator, setCurrentCreator] = useState<Creator | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
+  const { id } = useParams<CreatorProfileParams>();
+
+  const history = useHistory();
+
+  // Fetch Creator info from url query params
+  useEffect(() => {
+    setLoading(true);
+
+    CreatorAPIClient.getCreatorById(id)
+      .then((res) => {
+        if (res) {
+          setCurrentCreator(res);
+        }
+        setLoading(false);
+      })
+      .catch((error) => {
+        history.replace("/404");
+      });
+  }, [id, history]);
+
+  if (loading) {
+    return <LoadingSpinner />;
+  }
+
+  return currentCreator ? (
+    <div>
+      <>
+        <Box
+          minW="100vw"
+          minH="100vh"
+          bgImage={[null, null, background]}
+          bgRepeat="no-repeat"
+          backgroundSize="cover"
+          backgroundAttachment="scroll"
+          bgPosition="0 -120px"
+        >
+          <Center>
+            <Box w="80%">
```

Comment:
Above this line can you add `<Flex flexDirection="column" w="80%">` and remove the `w="80%"` from the Box. This will handle to layout of the other sections (Publications, Presentations)

Reply:
Handled these fixes quickly myself